module/VuFind/tests/bootstrap.php

```diff
@@ -31,8 +31,8 @@
 }
 
 // Make sure local config dir exists:
-if ($localDir = \VuFind\Config\Locator::getLocalConfigPath('', null, true)) {
-    if (!file_exists($localDir)) {
-        mkdir($localDir, 0777, true);
-    }
+$resolver = new \VuFind\Config\PathResolver();
```

Comment:
Using this in the bootstrapper seems to pose some chicken and egg problems. I wonder if this might actually be an instance where simply using the environment variable directly would actually make more sense. (Maybe not, just brainstorming...)

Reply:
Agreed. I changed it to check LOCAL_OVERRIDE_DIR and verify that it exists.

Reply:
Great, seems safer and more robust!